README.md

```diff
@@ -1,19 +1,56 @@
 # Shlifim Project - beyond 05 team 3
 
+
 ![shlifim_logo_2](https://user-images.githubusercontent.com/40122521/110316976-2990f880-8014-11eb-8ee8-96c21dd4b29f.png)
 
-Shlifim is the project our group will work on during the course.
+**Description**: Shlifim is the project our group will work on during the course.
 The project will be forum based web application for helping students find answers
 to their homework and assignments and also share summaries.
 
-The develpoment of this project will be in [Vagrant](https://www.vagrantup.com/) environment using [PipEnv](https://github.com/pypa/pipenv) and [Django](https://www.djangoproject.com/)
+[![Maintenance](https://img.shields.io/badge/Version%3F-Demo-yellow.svg)](https://github.com/beyond-io/shlifim/issues)
+[![Python Status](https://github.com/beyond-io/shlifim/actions/workflows/flake8.yml/badge.svg)](https://github.com/beyond-io/shlifim/actions/workflows/flake8.yml)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/beyond-io/shlifim/graphs/commit-activity)
+
+## Upcoming Features:
+* Register & Login 
+* Post a question
+* Watch asked questions
+* Answer a question
+* Up/Down vote an answer
+* Add tags to questions
+* Filter questions by topics / tags / most answered
+* Upload a file
+
+## Technology stack 
+* [Vagrant](https://www.vagrantup.com/) -  an open-source software product for building and maintaining portable virtual software development environments
+* [PipEnv](https://github.com/pypa/pipenv) - tool that manages a virtualenv for your projects
+* [Django](https://www.djangoproject.com/) - Python Web framework
 
-Team members:
-* Aviv Zafrani
-* Danit Levi
-* Ido Kahlon
-* Rebecca Tubman
-* Lior Noy
+
+## Dependencies
+The dependencies that must be installed for this software to work are:
+* [Vagrant](https://www.vagrantup.com/downloads)
+* [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
```

Comment:
I think you need to update this part with the dependencies in the `Pipfile`

Reply:
good point, I debated with myself whether to add those dependencies, but because Vagrant runs a bootstrap script that takes care of the download and installation of Pipenv, flake8, Python, and Django, I think the developer only need to download Vagrant and VirtualBox by himself and the rest of it Vagrant is taking care of.
Do you agree with me or should I still add those to the list in order to inform the developer what tools are being used?

Reply:
Ok, I understand the point. You can keep it like that.